CHANGELOG.md

```diff
@@ -314,8 +317,7 @@ v1.1.0
 
 - Use Go 1.22.3 for builds. (@kminehart)
 
-v1.0.0
-------
+## v1.0.0
```

Comment:
Did you mean to change this?

Reply:
This was not intentional, probably an unwanted format-on-save I suspect.